src/screens/FeedScreen/Footer.js

```diff
@@ -79,7 +79,7 @@ const styles = StyleSheet.create({
     paddingBottom: 16.12,
     paddingHorizontal: 20,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the borderBottomColor seems fine as long as `COLORS.gray100` is defined in your color constants and it meets the design requirements. Ensure that this color change doesn't affect the readability and overall UI consistency.

```diff
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->